tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -1473,4 +1477,155 @@ public function test_payment_gateway_disabled_for_unsupported_currency() {
 		);
 		$this->assertFalse( $this->wcpay_gateway->is_available_for_current_currency() );
 	}
+
+	public function currencies_for_minimum_accounts_provider() {
+		return [
+			[ 'usd', '$0.50', 50 ],
+			[ 'aed', '2.00 د.إ', 200 ],
```

Comment:
looks like a typo in aed currency.

Reply:
It's not a typo 😅 

This is caused by the RTL direction of the text, and I though there is a typo too. Then I saw that editors (VS Code, Atom, and TextEdit) are having issues formatting code like this. Turns out, there is no typo, and this is properly working code, even though it looks like it isn't. Not sure why, but it is fully functional, and works as expected.

Reply:
TIL, will read more about it.